strongname.cmd

```diff
@@ -0,0 +1,21 @@
+@echo off
+REM ******************************************
+REM ** 
+REM ** Locate folders and disable strongname 
+REM ** 
+REM ******************************************
+
+echo options: debug (default), release
+echo args: %1%
+
+set bconfig=debug
+if '%1' NEQ '' (set bconfig=%1%)
+
+set gotoFolder=adal\src\Microsoft.IdentityModel.Clients.ActiveDirectory\bin\%bconfig%\net45
+echo config: %gotoFolder%
+pushd %gotoFolder%
+
+sn.exe -Vr Microsoft.IdentityModel.Clients.ActiveDirectory.dll
+sn.exe -Vr Microsoft.Identity.Core.dll
+
```

Comment:
why are you disabling strongname for just this dll?

Reply:
that would enable the tests to run. Ideally it should be enough to have the dll name and the strongname identifier marked. I will continue testing... and perhaps will identify that I need to do more (so far this seems to work on my box).

Reply:
sounds good.